storage/replicate_queue.go

```diff
@@ -43,14 +57,17 @@ type replicateQueue struct {
 	gossip    *gossip.Gossip
 	allocator allocator
 	clock     *hlc.Clock
+	options   RebalancingOptions
```

Comment:
Then you don't store these here if you've already passed this options into the allocator.  All you have his is the case of aaNoop, in which case you can just have the rebalanceTarget deal with it.